### PR TITLE
Fix #943 - Added SumoMe compatibility

### LINKED ISSUE #943: Sumo Me plugin prevents settings page working correctly
When the Sumo Me plugin is active, the tabs on our page don't work properly. You have to refresh the page to get the content to load. Can we stop Sumo Me loading its assets on our screen as @arunbasillal did for some other plugins in the past? 
https://wordpress.org/plugins/sumome/
HS tags: `sumome`, `settings page`

### PR DESCRIPTION
**Pull request by arunbasillal**
https://github.com/wp-media/wp-rocket/issues/943

### CODE REVIEW

File: inc/3rd-party/plugins/sumome.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	 * Dequeue SumoMe styles
+	 *
+	 * @since 3.0.4
+	 * @author Arun Basil Lal
+	 */
+	function rocket_dequeue_sumo_me_css() {
+		
+		// Retun on all pages but WP Rocket settings page
+		$screen = get_current_screen();
+		if ( $screen->id != 'settings_page_wprocket' ) {
```
Comment (Tabrisrp) [inline inc/3rd-party/plugins/sumome.php]: need strict comparison here with !==

File: inc/3rd-party/plugins/sumome.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	 * Dequeue SumoMe inline script
+	 *
+	 * @since 3.0.4
+	 * @author Arun Basil Lal
+	 */
+	function rocket_dequeue_sumo_me_js() {
+		
+		// Retun on all pages but WP Rocket settings page
+		$screen = get_current_screen();
+		if ( $screen->id != 'settings_page_wprocket' ) {
```
Comment (Tabrisrp) [inline inc/3rd-party/plugins/sumome.php]: Same here

File: inc/3rd-party/plugins/sumome.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<?php 
+/**
+ * Compatibility with SumoMe
```
Comment (Tabrisrp) [inline inc/3rd-party/plugins/sumome.php]: We should add more details about the reason for this compatibility here, in case we forget why we did this 😄

Comment (arunbasillal) [reply inc/3rd-party/plugins/sumome.php]: All done :+1:

**Pull request merged**